changelog.d/pa-3262.fixed

```diff
@@ -0,0 +1,14 @@
+In expression-based languages, definitions are also expressions.
+
+This change allows dataflow to properly handle definition expressions.
+
+For example, the pattern `0 == 0` will match `x == 0` in
+
+```elixir
+def f(c) do
+  x = (y = 0)
```

Comment:
I'd hope we handled this already for C. 🤔

Reply:
I guess for C we may have an `Assign` there rather than a `VarDef`.

Reply:
Yeah, it's `Assign` in C, and for Ruby and Julia it seems to be `Assign` too.